tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1262,19 +1262,28 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
         const branch = "master"
         const path = "simple"
         const repoUrl = "https://github.com/rancher/fleet-examples"
+        let resourceCount = '18 / 18'
+        let sameResourceEachCluster = false
+        if (/\/2\.10/.test(Cypress.env('rancher_version'))) {
```

Comment:
What will happen in 2.9?

Reply:
I will expand it to the other versions. Nice catch.